### DISCUSSION TITLE
Add some drawing functions

### DISCUSSION THREAD

**Discussion by lkabuci** [codam-coding-college/MLX42#103] [Ideas]
in some of the graphic libraries like raylib they have some useful functions like drawing a line square.
will be a good idea to add them instead of recreating them every time.
* `mlx_draw_line(mlx_image_t *image, int xo, int yo, int x, int y)`
* `mlx_draw_square(mlx_image_t *image, int xo, int sideSize)`

**Comment by W2Wizard** [comment]
Too useful, sorry. The main goal of MLX is to basically be as simple and barebones as possible. These features are core tasks in other projects of 42 (fdf). They should have to research and implement these functions themselves.

The only additional features I may be able to accept are things that let you customize the window just for extra functionality (like Joystick support, drag and drop, ...) basically features outside the reach of the students to do it themselves but if they wish to explore and implement they can go the extra mile for it.

The goal of MLX itself is to just provide a simple canvas to draw on, nothing more. You should bring your own paint if you wish to draw something. This would also NEVER get accepted by 42 or any other campus currently using this library.
